applications/nrf_desktop/integration.rst

```diff
@@ -9,6 +9,9 @@ Integrating your own hardware
 
 This section describes how to adapt the nRF Desktop application to different hardware, and lists steps required for adding a new board.
 
+.. tip::
+   Make sure to get familiar with the :ref:`nrf_desktop_board_configuration_files` section before porting the application to a new board.
```

Comment:
@MarekPieta : Please confirm if the ref link is correct.

Reply:
I would add a new tag right above `Configuration` section and link there (would be more straightforward)